dockstore-webservice/generated/src/main/resources/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# the original file is dockstore/dockstore-webservice/src/main/resources/requirements/swagger-ui/requirements.template
```

Comment:
I don't see a requirements.template file.

Reply:
Ah, forgot to correct comment after I couldn't figure out how to rename the file while filtering